### ISSUE TITLE
textUpdateMode is applicable for entire survey, and not per question

### ISSUE THREAD

**Issue by ssbdel97** [surveyjs/survey-library#2294]
UPD (by Andrew):

---

The new property ``textUpdateMode`` is added into ``text`` and ``comment`` questions. For ``text`` question it is visible when ``inputType`` equals to ``text``.
There are three possible values, "default", "onBlur" and "onTyping". If the value is default then ``survey.textUpdateMode`` property is used.

---

### Are you requesting a feature, reporting a bug or asking a question?

Requesting a Feature/ Asking a Question

### What is the current behavior?

textUpdateMode is applicable for entire survey, and not per question

### What is the expected behavior?

Is there a way to set textUpdateMode per question? I have a usecase where I need the textUpdateMode mode to be different for different questions. For eg. I am trying to use inputmasking, but it does not work properly when textUpdateMode is set to "onTyping" but for other questions, the business logic demands it to be set to onTyping. Any work around this would be highly appreciated.

### How would you reproduce the current behavior (if this is a bug)?
https://plnkr.co/edit/fSLcFlLyiN3rdLzo

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

your_code_here

**Comment by ssbdel97** [comment]
Thank you for taking the time out implementing the feature. Really appreciate the quick turnaround time.
Can we also have this for dropdown questions having "Other" value? And if not, can we set it at global survey level, and allow for overriding in individual questions?

**Comment by andrewtelnov** [comment]
@ssbdel97 I am afraid "Other" value does not support onTyping at all. I guess we can support it on the global level. Do you have a real scenario when it should be set on a question level?

Thank you,
Andrew

**Comment by ssbdel97** [comment]
Hi, I am sorry for not specifying it earlier. I am using "Other" with "storeOthersAsComment" as false.
I have a usecase where I need to search something as user types in case of other value. But also in that very survey, I need to mask currency. If I use textUpdateMode on global survey level, the "Other" search use case works but not masking. I tried after update (v1.7.24), and it seems now textUpdateMode is not applicable for others when not storing them as comment.
Here is an implementation from v1.7.23, where if we set textUpdateMode (by design, it can be applied only at global survey level in this version), it does affect the "Other" value.

https://plnkr.co/edit/vcfLFvaZgS1iyAMm

If you will see, for each keystroke in other text box, the trigger function would be hit
